aws_jobs/cyhy-data-extract.py

```diff
@@ -26,16 +25,19 @@
 from ConfigParser import SafeConfigParser
 from datetime import datetime
 from dateutil.relativedelta import relativedelta
+import dateutil.tz as tz
```

Comment:
We generally try to separate the python imports at the top of the file into three groups:
* Built-ins
* 3rd-party dependencies
* Local imports

We also alphabetize the imports within each group.  Can you rearrange these accordingly?

Reply:
@mcdonnnj, my obsession with import organization is legendary among the dev team.